### ISSUE TITLE
macOS 10.15 (Catalina) relocates XQuartz X11R6 symlink, which cannot then be deleted

### ISSUE THREAD

**Issue by marcschwartz** [XQuartz/XQuartz#1]
Hi,

There is a post here:

https://discussions.apple.com/thread/250712783

that describes the basic issue with the installation/upgrade of Catalina and the relocation of XQuartz related files. As noted in the above post, the relocated files, if attempted to be moved to the Trash, cannot then be deleted. They cannot be deleted in place either via the CLI.

I get the same structure in my Trash:

![Screen Shot 2019-10-08 at 12 08 06 PM](https://user-images.githubusercontent.com/1749084/66416495-efa96600-e9cb-11e9-91c1-aadb48c6dd01.png)

There is a gist here:

https://gist.github.com/pwnsdx/d127873e24cef159d4d603accaf37ee4#file-gistfile1-txt

which has a series of CLI commands that suggest that this approach would be a way to remove XQuartz, then after a restart of the computer, would then presumably allow the aforementioned files in the Trash to be deleted.

However, this approach was not successful for me.

A Google search for other possible options did not show anything, which may not be a surprise, given that Catalina was just released yesterday.

Any guidance on resolving the underlying issue would be most appreciated, as presumably, anyone upgrading to Catalina, while using XQuartz, will face the same issue.

**Comment by marcschwartz** [comment]
Hi,

A quick follow up to note that since I posted the above, there are two additional comments in the Apple Community thread that I linked to above, one by 'faikbey' and one by me, regarding what appears to be a workaround for the problem, by booting into Recovery Mode, with some additional steps there, then rebooting into normal mode.

It would be helpful to get some guidance and/or better solutions to this issue.

Thanks.
